scipy/interpolate/tests/test_interpolate.py

```diff
@@ -993,6 +993,101 @@ def test_extrapolate_attr(self):
         # Testing extrapoation to actual function.
         xp_assert_close(y_ext, ak_true(x_ext), atol=1e-15)
 
+    @pytest.mark.skip('not fixed in current implementation')
```

Comment:
I'm two minds on whether adding skipped tests is useful. On the other hand, we can potentially consider these skipped tests as a kind of documentation, and the hard work to isolate the problematic cases has already been done.